cmd/cmd.go

```diff
@@ -60,6 +61,9 @@ func Execute() {
 // ErrExit writes an error to the given io.Writer & exits
 func ErrExit(w io.Writer, err error) {
 	log.Debug(err.Error())
+	if errors.Is(err, migrate.ErrMigrationSucceeded) {
+		printSuccess(w, "migration succeeded, re-run your command to continue")
```

Comment:
Add `os.Exit(0)` here? Otherwise looks like the err gets printed again. Also, return code should be 0 on success. Though I'm not sure if "the original command didn't run" should count as success?

Reply:
damn nice catch. I vote we exit with status 0 (success), a thing did succeed, and only because the user chose `Yes`. I've intentionally carved around a _non_ interactive version of this so we can write `qri connect --migrate` which will auto-migrate and continue